huaweicloud/services/elb/resource_huaweicloud_elb_logtanks.go

```diff
@@ -0,0 +1,144 @@
+package elb
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/elb/v3/logtanks"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func ResourceLogTanksV3() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceLogTanksV3Create,
+		ReadContext:   resourceLogTanksV3Read,
+		UpdateContext: resourceLogTanksV3Update,
+		DeleteContext: resourceLogTanksV3Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"loadbalancer_id": {
+				Type:     schema.TypeString,
```

Comment:
if this argument cant  be update, please set to forcenew

Reply:
done